Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1629,6 +1631,24 @@ void WebProcessProxy::updateAudibleMediaAssertions()
     }
 }
 
+void WebProcessProxy::updateMediaStreamingActivity()
+{
+    bool newMediaStreamingWebPage = WTF::anyOf(pages(), [] (auto& page) {
```

Comment:
s/newMediaStreamingWebPage/hasMediaStreamingWebPage/

Reply:
I copied the style of the method above `WebProcessProxy::updateAudibleMediaAssertions.
Will modify both